From 0f9f5096f722d20c6f21a7f4958502f8a705c92a Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Mon, 9 Mar 2015 17:40:07 +0800
Subject: [PATCH] tests: install test programs

--enable-install-test-programs allows cross compiling test programs
and installing them to $bindir.  This configure option propagates to
Makefile.am as HAVE_INSTALL_TESTS.

This was set for basic tests (modetest, etc.), but the tests/
directory was missed.

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
---
 tests/Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index cd11491..0c92175 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -65,4 +65,8 @@ endif
 
 check_PROGRAMS += $(TESTS)
 
+if HAVE_INSTALL_TESTS
+bin_PROGRAMS = ${TESTS}
+endif
+
 endif
-- 
2.2.0.rc0.207.ga3a616c

